docs/bookstore/page-0.5/pg0.5-env-setup.md

```diff
@@ -0,0 +1,214 @@
+
+!!! tip
+    Kuack suggests you to start from **/start**, write all the configuration files as you go over the tutorial, and check solutions when you get stuck.
+
+## **Shortcut**
+
+![Image](images/image10.png)
+
+Running `docs/code-samples/eventing/bookstore-sample-app/start/setup.sh` will automatically complete all tasks in this section.
+
+!!! warning
+
+    However, if you are not familiar with the process, we recommend reviewing the steps below.
+
+## **Instructions**
+
+### **Task 1: Set Up a Running Kubernetes Cluster with Knative Installed**
+
+![Image](images/image13.png)
+
+Please follow the instructions [here](https://knative.dev/docs/install/) to spin up your cluster with Knative installed!
+
+???+ success "Verify"
+
+    You should see the pods in the `knative-eventing` and `knative-serving` namespaces running before proceeding.
+
+    ![Image](images/image2.png)
+
+#### **Extra Step for Minikube Users:**
+
+![Image](images/image3.png)
+
+Attention! In case you're not using the Knative Quick Start, set up the tunnel manually to connect to services of type `LoadBalancer`:
+
+Run the following command and keep the terminal open:
+
+```shell
+minikube tunnel
+```
+
+???+ success "Verify"
+    If there aren't any error messages, it means you have set up the tunnel successfully.
+
+### **Task 2: Running the Bookstore Web App**
+
+![Image](images/image12.png)
+
+The Next.js frontend app is located in the `docs/code-samples/eventing/bookstore-sample-app/start/frontend` folder.
+
+Ensure that port 3000 on your local machine is not being used by another application.
+
+#### **Deploy the Frontend App**
+
+You can either [build the image locally](https://docs.docker.com/get-started/02_our_app/) or use our pre-built image. If you are using the pre-built image, you can proceed to the next step.
+
+When ready, run the following command to deploy the frontend app:
+
+```shell
+kubectl apply -f config/100-front-end-deployment.yaml
+```
+
+This will create the Deployment and expose it with a Service of type LoadBalancer to receive external traffic:
+
+![Image](images/image11.png)
+
+```shell
+kubectl get pods
+```
+
+You will see that your front end pod is running.
+
+![Image](images/image4.png)
+
+#### **Port Forwarding (Optional under condition)**
+
+![Image](images/image9.png)
+
+You might need to set up port forwarding to access the app from your local machine.
+
+Check if port forwarding is necessary by running:
+
+```shell
+kubectl get services -A
+```
+
+![Image](images/image1.png)
+
+If the `EXTERNAL-IP` for your frontend service is `127.0.0.1`, port forwarding is not needed.
+
+If port forwarding is required, open a new terminal and run:
+
+```shell
+kubectl port-forward svc/bookstore-frontend-svc 3000:3000
+```
+
+![Image](images/image8.png)
+
+Don’t close the terminal when port-forwarding is established.
+
+???+ success "Verify"
+
+    Visit [http://localhost:3000](http://localhost:3000) in your browser. The UI page should appear!
```

Comment:
I think if this link would open a new tab, it would make me easier to continue the tutorial